reuse/links.txt

```diff
@@ -1,7 +1,7 @@
 .. _Canonical website: https://canonical.com/
 .. _reStructuredText style guide: https://canonical-documentation-with-sphinx-and-readthedocscom.readthedocs-hosted.com/style-guide/
-.. _Sphinx reStructuredText Primer: https://tinyurl.com/rstprimer
+.. _Sphinx reStructuredText Primer: https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html
```

Comment:
This one was using tinyurl because it *should* fail the woke action - seems like there's something wrong with the action now, it shouldn't pass.